lib/tasks/groups.rake

```diff
@@ -6,7 +6,7 @@ namespace :db do
     Faker::Config.random = Random.new(42) # seeds the random number generator so Faker output is deterministic
     students = Student.all
     Assignment.all.each do |assignment|
-      num_groups = (assignment.short_identifier == 'A1' || assignment.short_identifier == 'A3') ? students.length : 15
+      num_groups = assignment.short_identifier == 'A1' || assignment.short_identifier == 'A3' ? students.length : 5
```

Comment:
Revert the changes in this file: 5 -> 15, but also how you fixed the Hound comment. Basically, this file shouldn't show any changes in the diff with master.